internal/proxy/metrics_info.go

```diff
@@ -50,12 +50,29 @@ func getQuotaMetrics() (*metricsinfo.ProxyQuotaMetrics, error) {
 			Rate:  rate,
 		})
 	}
+
+	getSubLabelRateMetric := func(label string) {
+		rates, err2 := rateCol.RateSubLabel(label, ratelimitutil.DefaultAvgDuration)
+		if err2 != nil {
+			err = err2
+			return
+		}
+		for s, f := range rates {
+			rms = append(rms, metricsinfo.RateMetric{
+				Label: s,
+				Rate:  f,
+			})
+		}
+	}
 	getRateMetric(internalpb.RateType_DMLInsert.String())
 	getRateMetric(internalpb.RateType_DMLUpsert.String())
 	getRateMetric(internalpb.RateType_DMLDelete.String())
 	getRateMetric(internalpb.RateType_DQLSearch.String())
+	getSubLabelRateMetric(internalpb.RateType_DQLSearch.String())
```

Comment:
Do we really need SubLabelRateMetric?

Why not simply sum all collections from this database on rootcoord?

Reply:
The current metric is the average of all collections, and detailed information about a collection cannot be obtained, so sub-labels are used to record more detailed information.